commit 1803f594cbf9bb2e662ac945038113d0d0cc5e89
Author: Johannes Berg <johannes.berg@intel.com>
Date:   Mon Jan 5 11:16:42 2015 +0100

    nl80211: document NL80211_BSS_STATUS_AUTHENTICATED isn't used
    
    The flag is no longer used (and hasn't been for a long time)
    since trying to track authentication (and make decisions based
    on state) was just causing issues all over - see commit
    95de817b9034d50860319f6033ec85d25024694c.
    
    Signed-off-by: Johannes Berg <johannes.berg@intel.com>

diff --git a/include/uapi/linux/nl80211.h b/include/uapi/linux/nl80211.h
index 18cb0aa06351..54f391141351 100644
--- a/include/uapi/linux/nl80211.h
+++ b/include/uapi/linux/nl80211.h
@@ -3265,6 +3265,9 @@ enum nl80211_bss {
 /**
  * enum nl80211_bss_status - BSS "status"
  * @NL80211_BSS_STATUS_AUTHENTICATED: Authenticated with this BSS.
+ *	Note that this is no longer used since cfg80211 no longer
+ *	keeps track of whether or not authentication was done with
+ *	a given BSS.
  * @NL80211_BSS_STATUS_ASSOCIATED: Associated with this BSS.
  * @NL80211_BSS_STATUS_IBSS_JOINED: Joined to this IBSS.
  *